proj/src/reducers/root.js

```diff
@@ -0,0 +1,13 @@
+import { combineReducers } from 'redux';
+
+import loginReducer from 'reducers/login/loginReducer'
+import projectsReducer from 'reducers/projects/projectsReducer'
+import statistics from 'reducers/statistics/statisticsReducer'
+import timeTrackingReducer from 'reducers/time-tracking/timeTrackingReducer'
+
+export default combineReducers({
```

Comment:
we need to talk about naming the reducers so we will avoid things like state.projects.projects in mapStateToProps